src/core/Akka.Remote/DeadlineFailureDetector.cs

```diff
@@ -28,13 +37,25 @@ public class DeadlineFailureDetector : FailureDetector
         /// heartbeats that will be accepted before considering it to be an anomaly.
         /// This margin is important to be able to survive sudden, occasional, pauses in heartbeat
         /// arrivals, due to for example garbage collect or network drop.</param>
-        /// <param name="clock">The clock, returning current time in milliseconds, but can be faked for testing
-        /// purposes. It is only used for measuring intervals (duration).</param>
-        public DeadlineFailureDetector(TimeSpan acceptableHeartbeatPause, Clock clock = null) : this(clock)
+        /// <param name="heartbeatInterval"></param>
+        /// <param name="clock">The clock, returning current time in milliseconds, but can be faked for testing purposes. It is only used for measuring intervals (duration).</param>
+        public DeadlineFailureDetector(
```

Comment:
Looks good to me - this constructor is only meant to be called internally by Akka.Remote / Cluster anyway.